packages/terra-core-docs/src/terra-dev-site/doc/divider/AccessibilityGuide.4.doc.mdx

```diff
@@ -0,0 +1,108 @@
+import DividerExampleAccess from "./example/DividerExampleAccess";
```

Comment:
```suggestion
import { Badge } from 'terra-divider/package.json?dev-site-package';
import DividerExampleAccess from "./example/DividerExampleAccess";

<Badge />
```

Reply:
added in https://github.com/cerner/terra-core/pull/3677/commits/90dbc52654717fe1c86bc3f5c6788289f093c9b3